### PR TITLE
Fix for when the 'migrate' command drops the custom default language

### PR DESCRIPTION
**Pull request by haalcala**
Fix for when the 'migrate' command drops the custom default language.

P.S. No ticket for this

### CODE REVIEW

**Comment by grundleborg** [comment]
@jfrerich looks like this doesn't compile.

**Comment by grundleborg** [comment]
Sorry, pinged the wrong person, meant to ping @haalcala :)

**Comment by haalcala** [comment]
Sorry, let me check again.

**Comment by hanzei** [comment]
@grundleborg @mgdelacroix I think this PR is ready for reivew.

File: cmd/mattermost/commands/config.go [hunk 1]
```diff
@@ -275,7 +276,15 @@ func configMigrateCmdF(command *cobra.Command, args []string) error {
...
 	to := args[1]
 
-	err := config.Migrate(from, to)
+	cwd, err := os.Getwd()
+
+	if err != nil {
+		return errors.Wrap(err, "failed to migrate config")
+	}
+
+	utils.InitTranslationsWithDir(path.Join(cwd, "i18n"))
```
Comment (mgdelacroix) [inline cmd/mattermost/commands/config.go:285]: As the `mattermost` command is always going to run from the server's directory, we can use the `TranslationsPreInit` function, for simplicity:

```suggestion
	if err := utils.TranslationsPreInit(); err != nil {
		return errors.Wrap(err, "failed to load translations while migrating config")
	}
```

**Review by mgdelacroix — COMMENTED** [review]
Thanks for the contribution @haalcala!! Apologies for the late review. The approach looks good and I've tested the fix and it's working correctly.

Suggested a change on the implementation.

**Comment by mgdelacroix** [comment]
Tried to update the pull request to merge on the `master` branch, but it is messing up the history. @haalcala could you please point the PR to the `master` branch?

**Comment by jasonblais** [comment]
@haalcala Thanks for your contribution! Let us know if you have any questions about the feedback above? Happy to help pointing the PR to the `master` branch if you're unsure about doing that.

**Comment by isacikgoz** [comment]
Gentle ping on this @haalcala :)

**Comment by haalcala** [comment]
> Gentle ping on this @haalcala :)

Ooppss. Sorry, been busy hacking your server codes. Let me have a look at it. :)

**Comment by haalcala** [comment]
Sorry, let me redo this patch on a later branch.  I'm migrating all my patches to 5.31 anyway.

**Pull request closed without merging**